tests/queries/0_stateless/01056_window_view_proc_hop_watch.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python3
-# Tags: no-parallel
```

Comment:
(I did actually not know that clickhouse-test can run Python tests 😄)

Reply:
I would be happier if it couldn't :smile: They are harder to run and harder to debug